implementations/rust/ockam/ockam_api/src/nodes/models/show.rs

```diff
@@ -0,0 +1,76 @@
+use ockam_multiaddr::{
```

Comment:
This whole file should be moved to `ockam_command`.

Reply:
Should I dump all its content (and the new serialization structs) to the `show.rs` file or do I create a new `model` module in `node` like is in `ockam_api` to store all this serialization logic?

Reply:
The current version has the models/show.rs version